neurom/core/_neuron.py

```diff
@@ -93,8 +112,9 @@ def iter_sections(neurites, iterator_type=Tree.ipreorder, neurite_filter=None):
         >>> n_points = [len(s.points) for s in iter_sections(pop,  neurite_filter=filter)]
 
     '''
-    return chain.from_iterable(iterator_type(neurite.root_node)
-                               for neurite in iter_neurites(neurites, filt=neurite_filter))
+    neurite_iter = NeuriteIter.NRN if iterator_type == Tree.inrnorder else NeuriteIter.Natural
```

Comment:
does passing `Tree.ipreorder` still work right?

Reply:
It still passes the tests so I guess so.